src/components/select/Select.js

```diff
@@ -190,6 +191,14 @@ export default class SelectComponent extends Field {
       label: label
     };
 
+    const skipOption = this.component.uniqueValues
+      ? !!this.selectOptions.find((selectOption) => _.isEqual(selectOption.value, option.value))
```

Comment:
This would not work for Resource select, and if they have custom objects in the list. Maybe we should only show this option if they are using the "Values" option?

Reply:
This is working for me with resource
![image](https://user-images.githubusercontent.com/47381348/82221062-295e2a00-9928-11ea-8bef-81283910c348.png)